plugin/src/features/Compilation/index.tsx

```diff
@@ -749,6 +749,7 @@ const Compilation: React.FC<CompilationProps> = ({ setAccordian }) => {
       // const get all values of allFiles object
       const allFilesValues = Object.values(allFiles)
 
+      console.log('allFiles', allFiles)
```

Comment:
Seems unnecessarily, perhaps just remove it.

Reply:
Oops, thought removed it. Thanks, good catch